package.json

```diff
@@ -78,19 +81,21 @@
     "@babel/runtime": "7.12.5",
     "ava": "1.2.1",
     "babel-plugin-macros": "2.8.0",
+    "eslint": "7.32.0",
     "eslint-config-prettier": "4.0.0",
     "husky": "4.3.0",
-    "mz": "2.7.0",
     "prettier": "1.16.4",
     "pretty-quick": "3.1.0",
     "react": "17.0.1",
     "react-dom": "17.0.1",
-    "semantic-release": "17.2.2",
-    "xo": "0.24.0"
+    "semantic-release": "17.2.2"
   },
   "peerDependencies": {
     "react": ">= 16.8.0 || 17.x.x || 18.x.x"
   },
+  "engines": {
+    "node": ">= 10"
```

Comment:
I think that we can drop support for 10 (and probably 12). What does Next.js support?

Reply:
switched to `node >= 12.0.0`, aligning with next.js